meteor/sofie-server/model/enums/error-code.ts

```diff
@@ -7,4 +7,6 @@ export enum ErrorCode {
 	NOT_ACTIVATED = 'NOT_ACTIVATED',
 	ALREADY_ACTIVATED = 'ALREADY_ACTIVATED',
 	RUNDOWN_IS_LOCKED = 'RUNDOWN_IS_LOCKED',
+	RUNDOWN_IS_ACTIVE = 'RUNDOWN_IS_ACTIVE',
```

Comment:
Would be nice if this was `RUNDOWN_ALREADY_ACTIVE` as being active is not an Error. Another Rundown _already_ being active is the error :)

Reply:
A rundown being active IS an error, if you are trying to delete it, as we aggred that an active rundown couldn't be deleted.

Might need a talk about this, to figure out the correct naming.

Reply:
To me it sounds like those are two different errors.

Reply:
We don't want to manage 100+ errors.

`ALREADY_ACTIVATED` sounds a lot like `RUNDOWN_IS_ACTIVE` to me, so maybe we should delete `ALREADY_ACTIVATED` and use `RUNDOWN_IS_ACTIVE` for that scenario?

I am also fine with keeping both, but we need to draw a line somewhere with how specific we want our error types to be :)

Reply:
As both currently evaluates to a `BAD_REQUEST`, i am open to the idea of using the `RUNDOWN_IS_ACTIVE` in that scenario.

Should i make that change in this PR, or are we doing that in a seperate on?